actioncable/lib/action_cable/server/configuration.rb

```diff
@@ -29,7 +29,16 @@ def channel_class_names
           Pathname.new(channel_path).basename.to_s.split('.').first.camelize
         end
       end
+
+      # Returns constant of subscription adapter specified in config/cable.yml
+      # If the adapter cannot be found, this will default to the Redis adapter
+      def subscription_adapter
+        # Defaults to redis if no adapter is set
+        adapter = cable.fetch('adapter') { 'redis' }
+        adapter = adapter.camelize
+        adapter = "PostgreSQL" if adapter == "Postgresql"
```

Comment:
Thanks for taking me to school, all good :+1:

> Den 16. jan. 2016 kl. 16.39 skrev Jon Moss <notifications@github.com>:
> 
> In actioncable/lib/action_cable/server/configuration.rb <https://github.com/rails/rails/pull/22950#discussion_r49933190>:
> 
> > @@ -29,7 +29,16 @@ def channel_class_names
> >            Pathname.new(channel_path).basename.to_s.split('.').first.camelize
> >          end
> >        end
> > +
> > +      # Returns constant of subscription adapter specified in config/cable.yml
> > +      # If the adapter cannot be found, this will default to the Redis adapter
> > +      def subscription_adapter
> > +        # Defaults to redis if no adapter is set
> > +        adapter = cable.fetch('adapter') { 'redis' }
> > +        adapter = adapter.camelize
> > +        adapter = "PostgreSQL" if adapter == "Postgresql"
> I think we went with "PostgreSQL" to have it match AR conventions. AR does the same thing, here <https://github.com/rails/rails/blob/master/activerecord/lib/active_record/connection_adapters/connection_specification.rb#L37>
> —
> Reply to this email directly or view it on GitHub <https://github.com/rails/rails/pull/22950/files#r49933190>.
> 

--
Kasper